# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/layout.js
# Commit: 6591f14bbc71
# Full Hash: 6591f14bbc713f11b6316712d931ecc19a443a5c
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-05 03:52:32
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 06dda9072cb7 -r 6591f14bbc71 accessible/tests/mochitest/layout.js
--- a/accessible/tests/mochitest/layout.js	Mon Oct 04 19:28:49 2021 +0000
+++ b/accessible/tests/mochitest/layout.js	Mon Oct 04 19:46:54 2021 +0000
@@ -280,26 +280,26 @@
 
   info(`DPR is: ${aDPR}`);
   isWithin(
+    xInCSS.value,
     x.value / aDPR,
-    xInCSS.value,
     1,
     "X in CSS pixels is calculated correctly"
   );
   isWithin(
+    yInCSS.value,
     y.value / aDPR,
-    yInCSS.value,
     1,
     "Y in CSS pixels is calculated correctly"
   );
   isWithin(
+    widthInCSS.value,
     width.value / aDPR,
-    widthInCSS.value,
     1,
     "Width in CSS pixels is calculated correctly"
   );
   isWithin(
+    heightInCSS.value,
     height.value / aDPR,
-    heightInCSS.value,
     1,
     "Height in CSS pixels is calculated correctly"
   );